### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by telezhnaya**
Done:
- Changed the structure. The main page is much more lightweight but still reflects the most important topics - please comment on what should be added/deleted related to the structure
- Carefully moved the pieces to other files, you don't need to re-read them, I haven't changed anything
- First version of the instruction for the users

Some thoughts:
- People may be confused with `NEAR Stake Wars IV` VS `StatelessNet`, there are many places where we can correctly use both. Instead of explaining the difference, I prefer to stick to one naming where possible, and use the other only if it's necessary in the context. I suggest to stick to `NEAR Stake Wars IV` at least at the main page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,21 @@
...
 
-## What are the upcoming changes?
-In Q2 2024, NEAR protocol will introduce the biggest upgrade since its mainnet launch in how the protocol operates, Stateless validation. The upgrade includes the following features:
-* Stateless validation: Validators do not have to track all shards anymore and can retrieve everything they need to validate state change (state witness) over the network. With stateless validation, NEAR can finally achieve true form of sharding and hardware requirements for a validator node can be dramatically reduced. State validation does not require local tracking of any shards as nodes can retrieve everything they need to validate state change over the network. As a result, hardware requirements for a validator node can be dramatically reduced.
-* In memory trie: Validator nodes can load the entire state into RAM for maximized performance. With minimized storage access, transaction throughput will be greatly improved. 
+You can find the stats [here](https://nearinc.grafana.net/public-dashboards/ebf4227a92de4bb5b3e02938129be6ae).
```
Comment (gmilescu) [inline README.md:5]: As per our discussion, let's include the dashboard next week after the network is up and running and is stabilised.

**Review by gmilescu — CHANGES_REQUESTED** [review]
Overall look good, thanks for putting it together. Please see the comment I left around the dashboard.

File: HOW_TO.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+## How to join the network?
+
+RPC address: TODO
```
Comment (telezhnaya) [inline HOW_TO.md:3]: @andrei-near

File: HOW_TO.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+We recommend you to use [near-cli documentation](https://docs.near.org/tools/near-cli), e.g. how to [create an account](https://docs.near.org/tools/near-cli#near-create-account).
+
+StatelessNet is a sandbox created for load testing purposes.
+The load from mainnet may be forwarded to StatelessNet.
+It means that all the existing mainnet accounts would be already occupied.
+It will not affect the activity on mainnet in any case.
+
+If you need some tokens for your activity, fill out the form: TODO.
```
Comment (telezhnaya) [inline HOW_TO.md:12]: @gmilescu

File: HOW_TO.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## How to become a validator?
+
+The most recent release of StatelessNet: https://github.com/near/nearcore/commit/ef5564c4950d448144d23488df8b63d45e6b0209
+
+You need to build it with the additional argument
+```
+--features statelessnet_protocol
+```
+
+Sample config: TODO
```
Comment (telezhnaya) [inline HOW_TO.md:23]: @wacban

File: HOW_TO.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+In order to be able to create staking pool, you need some tokens.
+Please fill in the validator form: TODO
```
Comment (telezhnaya) [inline HOW_TO.md:26]: @gmilescu

File: HOW_TO.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+### Hardware requirements
+
+Assuming the heaviest setup where a node tracks all shards and stores all shards in memory:
+- 1TB disk
+- 32GB RAM
+- 8 cores
+
+TODO: add the setup for tracking only 1 shard. We promote StatelessNet as an ability to have more light-weighted validators, we need to show this in practice.
```
Comment (telezhnaya) [inline HOW_TO.md:35]: @wacban

**Review by telezhnaya — COMMENTED** [review]
Asking for some help here and there

File: HOW_TO.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+## How to join the network?
+
+RPC address would be provided here shortly.
+
+We recommend you to use [near-cli documentation](https://docs.near.org/tools/near-cli), e.g. how to [create an account](https://docs.near.org/tools/near-cli#near-create-account).
+
+StatelessNet is a sandbox created for load testing purposes.
+The traffic from mainnet may be forwarded to StatelessNet.
```
Comment (wacban) [inline HOW_TO.md:8]: nit: The StatelessNet will be initiated with a copy of mainnet state adjustments to allow the dev full control of the newtork. In the future the protocol team may enable mirroring mainnet traffic in StatelessNet.

File: HOW_TO.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+It means that all the existing mainnet accounts would be already occupied.
+It will not affect the activity on mainnet in any case.
+
+If you need some tokens for your activity, fill out the form: [the link to the form will be provided shortly].
```
Comment (wacban) [inline HOW_TO.md:12]: Either this or we will provide a website to do that.

File: HOW_TO.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+## How to become a validator?
```
Comment (wacban) [inline HOW_TO.md:14]: I would subdivide into three sections
* how to build the binary
* how to get the config and genesis
* how to get the backup 

And I would also add a link to the page with instructions for setting up a mainnet validator

File: HOW_TO.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+The most recent release of StatelessNet: https://github.com/near/nearcore/commit/ef5564c4950d448144d23488df8b63d45e6b0209
+
+You need to build it with the additional argument
+```
+--features statelessnet_protocol
+```
+
+Sample config: [will be provided shortly]
+
+In order to be able to create staking pool, you need some tokens.
```
Comment (wacban) [inline HOW_TO.md:25]: I thought we decided to not use staking pools unless a need arises? 
I would rephrase to "In order to become a validator.. "

File: HOW_TO.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+Please fill in the validator form: [the link to the form will be provided shortly].
+
+### Hardware requirements
+
+Assuming the heaviest setup where a node tracks all shards and stores all shards in memory:
+- 1TB disk
+- 32GB RAM
+- 8 cores
+
+We will add the requirements for tracking one shard shortly.
```
Comment (wacban) [inline HOW_TO.md:35]: Shortly is relative, we are not yet ready to enable it (it's not tested). I would say that in later stages of StatelessNet we are planning on enabling single shard tracking and that instructions will be shared then.

File: README.md [hunk 12]
```diff
@@ -1,62 +1,16 @@
-# NEAR Protocol Stateless Validation available for public preview
+# NEAR Stake Wars IV: Attack of Transactions
 
-NEAR Protocol is preparing a major update of the core protocol architecture that takes scalability to the next level. This update is called stateless validation and it has ample impact on how the network operates internally. We invite everyone to join an early access network that runs on this new architecture, StatelessNet, and run a stateless validator node or test the network throughput and scalability.
+NEAR Protocol is preparing a major update of the core protocol architecture that takes scalability to the next level. This update is called stateless validation, and it has ample impact on how the network operates internally. We invite everyone to join an early access network that runs on this new architecture, StatelessNet, and run a stateless validator node or test the network throughput and scalability.
```
Comment (wacban) [inline README.md:3]: We only invite mainnet validators to be statelessnet validators but everyone to bug bounty and traffic generation.

File: TIMELINE.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+We plan two stages, each with different focus areas:
+* Stage 1, February 2024: StatelessNet will focus on demonstrating functional readiness of upcoming changes. MVP version of stateless validation with minimal optimization will be released first and followed by iterations on feature completeness.
+* Stage 2, March 2024: Once all critical features for stateless validation are stabilized and corner cases are addressed, StatelessNet will move on to the next stage, focusing on performance and reliability optimization. Multiple rounds of load tests and stress tests will be conducted to test the limit of stateless validation and identify bottlenecks. Engineering focus will be around optimizations on performance and reliability.
+
+## Notable dates
+Here are couple notable (approximate) dates during duration of the program
+* February 1st: Stake Wars IV Stage 1 starts
+* February 14th: Bug bounty reward program starts
+* Feb 29th - March 3rd: NEAR Protocol shares Stateless Validation and Stake Wars with the blockchain community at EthDenver.
+* March 1st: Sake Wars IV Stage 2 starts. Reward programs start for traffic generation and running nodes.
```
Comment (wacban) [inline TIMELINE.md:12]: typo Sake Wars
I would totally participate in that too though ;)

File: TECHNICAL_DETAILS.md [hunk 14]
```diff
@@ -0,0 +1,13 @@
+## What are the upcoming changes?
+In Q2 2024, NEAR protocol will introduce the biggest upgrade since its mainnet launch in how the protocol operates, Stateless validation. The upgrade includes the following features:
+* Stateless validation: Validators do not have to track all shards anymore and can retrieve everything they need to validate state change (state witness) over the network. With stateless validation, NEAR can finally achieve true form of sharding and hardware requirements for a validator node can be dramatically reduced. State validation does not require local tracking of any shards as nodes can retrieve everything they need to validate state change over the network. As a result, hardware requirements for a validator node can be dramatically reduced.
```
Comment (wacban) [inline TECHNICAL_DETAILS.md:3]: nit: state change -> the state transition
I would also not mention the state witness, it's an implementation detail.

nit: you wrote "hardware requirements for a validator node can be dramatically reduced." twice

File: TECHNICAL_DETAILS.md [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+* In memory trie: Validator nodes can load the entire state into RAM for maximized performance. With minimized storage access, transaction throughput will be greatly improved.
```
Comment (wacban) [inline TECHNICAL_DETAILS.md:4]: The "greatly" is still under evaluation. In-memory-trie reduces the storage access latency but then other things become bottlenecks and the total improvement isn't as great as just the `memory speed` / `disk speed`

File: TECHNICAL_DETAILS.md [hunk 16]
```diff
@@ -0,0 +1,13 @@
...
+
+The upgrade will be transparent and blockchain users should not experience any downtime and no special treatment from validators will be required, apart from adjusting the hw specs to the new node roles. Once stateless validation is in place, it will open the door to a series of additional network performance and scalability improvements.
```
Comment (wacban) [inline TECHNICAL_DETAILS.md:6]: nit: hw -> hardware
also this is the first time you mention new roles, I would leave it out completely for now or give a proper explanation somewhere

File: HOW_TO.md (hunk 9 above)
Comment (wacban) [reply HOW_TO.md:14]: Also giving a commit seems a bit weird, perhaps we can at least tag it or create a branch for it? WDYT?

**Review by wacban — COMMENTED** [review]
Thanks for looking into it, I have a few comments, but overall looks great! I'll let @gmilescu finish the review.

File: README.md (hunk 12 above)
Comment (telezhnaya) [reply README.md:3]: I don't think we should limit being a validator only to existing validators. OFK current validators are our main audience, but we don't plan to restrict anyone trying a new role

File: TIMELINE.md (hunk 13 above)
Comment (telezhnaya) [reply TIMELINE.md:12]: We should organise them as well for sure

File: TECHNICAL_DETAILS.md (hunk 15 above)
Comment (telezhnaya) [reply TECHNICAL_DETAILS.md:4]: We emphasise our intentions here, I think such wording is fine

File: HOW_TO.md (hunk 9 above)
Comment (telezhnaya) [reply HOW_TO.md:16]: Agree with everything!
I created the tag, added the tag and the link to the validator docs to the landing.
I like the idea to split the details into 3 sections you suggested, but I need your help because I don't have enough context to fill in each section

File: HOW_TO.md [hunk 17]
```diff
@@ -0,0 +1,53 @@
+## How to join the network?
+
+RPC address would be provided here shortly.
+
+We recommend you to use [near-cli documentation](https://docs.near.org/tools/near-cli), e.g. how to [create an account](https://docs.near.org/tools/near-cli#near-create-account).
```
[the remark this answers is not in the document]
Comment (telezhnaya) [reply HOW_TO.md]: Yes, it's still unresolved
Rephrased the doc

File: README.md (hunk 12 above)
Comment (wacban) [reply README.md:3]: There is a lot of confusion around this, please see this thread for hopefully the latest status:
https://near.zulipchat.com/#narrow/stream/422293-pagoda.2Fcore.2Fstake-wars-iv/topic/Who's.20eligible.20for.20rewards/near/419242181

**Review by jancionear — COMMENTED** [review]
I think it would be good to change the `Traffic generation` header to something else, as currently it suggests that only existing mainnet validators are eligible for rewards for traffic generation.
Maybe `High uptime nodes`?

File: HOW_TO.md (hunk 9 above)
Comment (wacban) [reply HOW_TO.md:16]: * how to build the binary - Can you check what is the recommended way in the public documentation and then just add the sv feature to it? I would expect it to like something like this: 
```cargo build -p neard --release --features statelessnet_protocol```
* config and genesis - @andrei-near I have no clue but I hope @andrei-near will put it in a cloud and give us the links
* backup - @andrei-near again :)

**Review by gmilescu — APPROVED** [review]

File: HOW_TO.md (hunk 9 above)
Comment (jancionear) [reply HOW_TO.md:16]: How would updating to the latest version work with tags?
I'm no git expert, but I read that there can be only one tag with some name. Are there going to be multiple tags for subsequent versions?

I think it would be more user friendly to have a branch. Then the user can just run `git pull origin stake-wars-iv` to get the latest version (Or have a script do it every X hours)

**Comment by telezhnaya** [comment]
Merging this as it is, I'll collect all the tricky parts and open a new PR

File: HOW_TO.md (hunk 9 above)
Comment (telezhnaya) [reply HOW_TO.md:16]: @jancionear tag is just a pointer. We can change the tag as well (drop and recreate it to the desired commit)

File: README.md (hunk 12 above)
Comment (telezhnaya) [reply README.md:3]: I re-checked, it's covered in bounty page. Anyone can still run a validator node, but only current mainnet validators may receive the reward
